var/spack/repos/builtin/packages/h5utils/package.py

```diff
@@ -12,10 +12,18 @@ class H5utils(AutotoolsPackage):
 
     homepage = "http://ab-initio.mit.edu/wiki/index.php/H5utils"
     url      = "http://ab-initio.mit.edu/h5utils/h5utils-1.12.1.tar.gz"
```

Comment:
You can have the main URL match the new download location using the following:

```suggestion
    url      = "https://github.com/NanoComp/h5utils/releases/download/1.13.1/h5utils-1.13.1.tar.gz"
```